android-components/components/ui/tabcounter/src/main/res/layout/mozac_ui_tabcounter_layout.xml

```diff
@@ -39,7 +39,8 @@
 
         <androidx.appcompat.widget.AppCompatImageView
             android:id="@+id/counter_mask"
-            app:srcCompat="@drawable/mozac_ui_tabcounter_badge"
+            app:srcCompat="@drawable/mozac_ic_private_mode_circle_fill_20"
+            app:tint="@color/photonPurple60"
```

Comment:
The whole image appears as a solid purple from my testing.

Reply:
I suspect you want to do something similar to https://github.com/mozilla-mobile/firefox-android/pull/3897/files#diff-0e84228aedfe52ad8170c30ca47e19702b7052925aad961f2ffe8a585c997a59R11

Reply:
whoops you're right! I fixed the color